tests/installation/partitioning_smalldisk_storageng.pm

```diff
@@ -29,7 +29,7 @@ sub run {
         select_console 'root-console';
     }
     my $lsblkcmd = q/echo "[$(lsblk -n -l -o SIZE,NAME -d -e 7,11,254 -b | sort -n | awk '(NR == 1) {print $2}')]"/;
-    $lsblkcmd = q/echo "[$(lsblk -n -l -o SIZE,NAME,TYPE -e 7,11,254 -b | sort -n | awk '($3 == "mpath" && NR == 1) {print $2}')]"/
+    $lsblkcmd = q/echo "[$(lsblk -n -l -o SIZE,NAME,TYPE -e 7,11 -b | grep 'mpath' | sort -n | awk '(NR == 1) {print $2}')]"/
```

Comment:
I do not see any difference in the output of `lsblk -n -l -o SIZE,NAME,TYPE -e 7,11 -b | grep 'mpath' | sort -n | awk '(NR == 1) {print $2}'` with and without 254.

In my opinion the actual fix came from moving the mpath match outside of the `awk` command.

Could you try with: `lsblk -n -l -o SIZE,NAME,TYPE -e 7,11,254 -b | grep 'mpath' | sort -n | awk '(NR == 1) {print $2}'` to confirm?

In the current installation it seems to work:
```
blackcurrant-1:~ # lsblk -n -l -o SIZE,NAME,TYPE -e 7,11,254 -b | grep 'mpath' | sort -n | awk '(NR == 1) {print $2}'
SAIX_VDASD_0008202a00007a000000016e947bd5b6.5
```
But we would need to confirm how it behaves during installation after the `wipefs` command.

Reply:
`wipefs` command is executed at beginning of installation before launching yast/installer, at this time the multipath is not active, when running partitioning_smalldisk_storageng.pm, multipath device is active. Using the change in commit#007d818, partitioning_smalldisk_storageng.pm cannot find correct multipath device, please refer to http://openqa.qa2.suse.asia/tests/40362#step/partitioning_smalldisk_storageng/5.

Reply:
Indeed, but I think it failed there not due to the 254 inclusion in `lsblk`, but due to not all partitions being wiped with `wipefs` and `lsblk` listing some partitions in its output (type == part), probably because as you say, multipath was not enabled when `wipefs` was executed.

I think:
```
lsblk -n -l -o SIZE,NAME,TYPE -e 7,11,254 -b | grep 'mpath' | sort -n | awk '(NR == 1) {print $2}'
```
Should work just as well as:
```
lsblk -n -l -o SIZE,NAME,TYPE -e 7,11 -b | grep 'mpath' | sort -n | awk '(NR == 1) {print $2}'
```
With the added benefit that you exclude zram devices as well.

As I said: in my opinion the actual fix is moving the `mpath` pattern matching outside of `awk` into `grep`, not the `-e 254` part.

Reply:
Got it, thanks! Keep '254' and move "mpath" out of awk, it works now.
`blackcurrant-1:~ # lsblk -n -l -o SIZE,NAME,TYPE -e 7,11,254 -b | sort -n | grep "mpath" | sort -n | awk '(NR == 1) {print $2}'
SAIX_VDASD_0008202a00007a000000016e947bd5b6.5`

Reply:
Verified on 15SP3, it work as well: `0:blackcurrant-1:~ # lsblk -n -l -o SIZE,NAME,TYPE -e 7,11,254 -b | sort -n | grep "mpath" | sort -n | awk '(NR == 1) {print $2}'`
`SAIX_VDASD_0008202a00007a000000016e947bd5b6.5`

Reply:
The code is updated, thank you all!